client/src/plugins/notification.js

```diff
@@ -0,0 +1,5 @@
+import Notifications from 'vue-notification';
```

Comment:
move this just to main.js

I think in summary there will be less lines of code